src/ParticipationStorage/ParticipationStorageInterface.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Phpab\Phpab\Storage;
+
+/**
+ * Stores the participation state of the user
+ */
+interface ParticipationStorageInterface
```

Comment:
ha, i see both of your comment :-D Long running ajax request maybe.

- Analytics/StorageInterface
- Participation/StorageInterface
- Participation/FilterInterface

seems like a good fit to me